nibabel/nicom/tests/test_dicomreaders.py

```diff
@@ -40,7 +40,10 @@ def test_passing_kwds():
     # Check that we correctly pass keywords to dicom
     dwi_glob = 'siemens_dwi_*.dcm.gz'
     csa_glob = 'csa*.bin'
-    import dicom
+    try:
+        import dicom as pydicom
+    except ImportError:
+        import pydicom
```

Comment:
Is it worth importing `InvalidDicomError` here, like you did with `read_file`? I know there's no change in the package path, but would be more consistent with the others (including `Dataset`)